include/model/create_document.js

```diff
@@ -106,18 +105,6 @@ module.exports = function DocumentCreatorModule(pb) {
     };
 
     /**
-     * Formats usernames
-     *
-     * @method usernameFormatting
-     * @param {Object} post Key value pair object
-     */
-    DocumentCreator.usernameFormatting = function(post){
-        if(util.isString(post['username'])) {
-            post['username'] = post['username'].toLowerCase();
-        }
-    };
```

Comment:
Got it.  We plan to move the whole "DocumentCreator" prototype.  It doesn't serve much of a purpose.  I would suggest implementing the fix such that it doesn't rely on the document creator.  Two suggestions: either use the UserService to create the user or call the individual functions from the create method.